### ISSUE TITLE
generate more predictions with more trunk samples and diffusion samples

### ISSUE THREAD

**Issue by liwei-c** [chaidiscovery/chai-lab#237]
Hi Chai-lab team, many thanks for making the model open source! It was mentioned the [technical report](https://www.biorxiv.org/content/10.1101/2024.10.10.615955v1), that you run 25 predictions from 5 trunk samples and 5 diffusion samples. I can see that the number of diffusion samples are defined as 5, but could you explain where does the 5 trunk samples come from? Perhaps it's by setting different seeds?

**Comment by wukevin** [comment]
> Hi Chai-lab team, many thanks for making the model open source! It was mentioned the [technical report](https://www.biorxiv.org/content/10.1101/2024.10.10.615955v1), that you run 25 predictions from 5 trunk samples and 5 diffusion samples. I can see that the number of diffusion samples are defined as 5, but could you explain where does the 5 trunk samples come from? Perhaps it's by setting different seeds?

Yes - by setting different seeds you can get different trunk samples, so running with 5 different seeds, each generating 5 diffusion samples, will get you 25 different outputs.

Happy folding!
